futures/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lapin-futures"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
Please don't update the version number, it will get updated when we release a new version